versions/3.1.1.md

```diff
@@ -314,12 +314,22 @@ This is the root object of the [OpenAPI document](#openapi-description).
-| <a name="oas-security"></a>security                    |  [[Security Requirement Object](#security-requirement-object)]  | A declaration of which security mechanisms can be used across the API. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. Individual operations can override this definition. To make security optional, an empty security requirement (`{}`) can be included in the array.                                                                                                                                               |
+| <a name="oas-security"></a>security                    |  [[Security Requirement Object](#security-requirement-object)]  | See [the `security` field](#oas-security-field).
 | <a name="oas-tags"></a>tags                            |                  [[Tag Object](#tag-object)]                   | A list of tags used by the document with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools. Not all tags that are used by the [Operation Object](#operation-object) must be declared. The tags that are not declared MAY be organized randomly or based on the tools' logic. Each tag name in the list MUST be unique.                                                                                                                                                                               |
 | <a name="oas-external-docs"></a>externalDocs            | [External Documentation Object](#external-documentation-object) | Additional external documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+###### <a name="oas-security-field"></a>The `security` field
+
+The `security` field describes how requests are authorized. If omitted, then requests are always authorized; otherwise, it is an array of security requirement objects, only one of which needs to be satisfied for the request to be authorized.
```

Comment:
What does
> then requests are always authorized
mean?

Top-of-mind interpretations:
1. Requests need authorization, and I don't tell you how, or
2. Requests don't need authorization

I'd argue that it means the first, because the second can be made explicit via `security: [{}]`.

Reply:
If you mean "what was the author's intention behind those words", then I can answer authoritatively: it means #2.

However, I can see how it could be interpreted the other way. Do you have a suggested alternative wording?